### PR TITLE
Fixed syntax highlighting errors causing red blocks to render.

### PR DESCRIPTION
**Pull request by othercorey**
I noticed a number of pages had these red outline boxes rendering in some of the code examples.  It seems to be some sort of syntax highlighter error output.

This corrects several unterminated string constants in php examples.

This also sets the code block language to bash for a majority of the console examples.  I added a .. code-block:: prefix for all of these.

This also fixes a couple random document errors like the file moved link in chronos.rst and some random unterminated inline formatters that I ran across checking output.

### CODE REVIEW

File: en/console-and-shells/option-parsers.rst [hunk 1]
```diff
@@ -2,7 +2,9 @@ Option Parsers
 ##############
 
 .. php:namespace:: Cake\Console
-.. php:class:: ConsoleOptionParser
+.. php:class
+
+.. code-block:: bashOptionParser
```
Comment (othercorey) [inline en/console-and-shells/option-parsers.rst]: I didn't notice this change.  This is unexpected and will revert.

File: en/console-and-shells/server-shell.rst [hunk 2]
```diff
@@ -20,7 +22,9 @@ terminal.
 Changing the Port and Document Root
 ===================================
 
-You can customize the port and document root using options::
+You can customize the port and document root using options
```
Comment (markstory) [inline en/console-and-shells/server-shell.rst]: Shouldn't these paragraphs end with a `.` or a `:`?

Comment (othercorey) [reply en/console-and-shells/server-shell.rst]: I see, you're right.  I missed that the :: would generate a : at the end of the sentence.

File: tr/console-and-shells.rst [hunk 3]
```diff
@@ -485,7 +489,7 @@ Kanca metodları
 Seçenekleri Yapılandırmak ve Yardım Üretmek
 ===========================================
 
-.. php:class:: ConsoleOptionParser
+.. php:class OptionParser
```
Comment (dereuromark) [inline tr/console-and-shells.rst]: are you sure those changes are correct?

Comment (othercorey) [reply tr/console-and-shells.rst]: Yea, that was caused by me.  I missed reverting a couple places when squashing earlier.  Not fully verifying every regex change can be dangerous!

Comment (othercorey) [reply tr/console-and-shells.rst]: Hmm, no, sorry, I don't know what this specific change is.  I'm not sure how I would have generated that.  sigh.

File: fr/console-and-shells.rst [hunk 4]
```diff
@@ -688,7 +694,7 @@ Méthodes Hook
 Configurer les options et générer de l'aide
 ===========================================
 
-.. php:class:: ConsoleOptionParser
+.. php:class OptionParser
```
Comment (othercorey) [inline fr/console-and-shells.rst]: Here as well.  Needs to be reverted.

File: ja/console-and-shells/input-output.rst [hunk 5]
```diff
@@ -2,7 +2,7 @@
 ####################
 
 .. php:namespace:: Cake\Console
-.. php:class:: ConsoleIo
+.. php:class Io
```
Comment (othercorey) [inline ja/console-and-shells/input-output.rst]: This too.

File: ja/console-and-shells/option-parsers.rst [hunk 6]
```diff
@@ -2,7 +2,7 @@
 ##################
 
 .. php:namespace:: Cake\Console
-.. php:class:: ConsoleOptionParser
+.. php:class OptionParser
```
Comment (othercorey) [inline ja/console-and-shells/option-parsers.rst]: This too.

File: pt/console-and-shells.rst [hunk 7]
```diff
@@ -473,7 +477,7 @@ ambiente ``ANSICON`` esteja presente.
 Opções de configuração e Geração de ajuda
 =========================================
 
-.. php:class:: ConsoleOptionParser
+.. php:class OptionParser
```
Comment (othercorey) [inline pt/console-and-shells.rst]: This too.

File: tl/console-and-shells.rst [hunk 8]
```diff
@@ -657,7 +663,7 @@ Hook Methods
 Configuring Options and Generating Help
 =======================================
 
-.. php:class:: ConsoleOptionParser
+.. php:class OptionParser
```
Comment (othercorey) [inline tl/console-and-shells.rst]: This too.

File: tr/console-and-shells.rst (hunk 3 above)
Comment (othercorey) [reply tr/console-and-shells.rst]: I know what it is now. I had originally marked them with .. code-block:: console but changed it to .. code-block:: bash.

I won't be making large changes like that again :)

**Pull request closed without merging**